source/client/ui/story/styles.scss

```diff
@@ -96,6 +98,10 @@ ff-tab-header, ff-dock-panel-header {
     height: 28px !important; 
     margin: 8px;
     align-self: center;
+
```

Comment:
Please remove - per discussion comment, this makes functional sense but should not be tied to  a specific width.